server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/FilterToSearchQuery.java

```diff
@@ -36,7 +36,13 @@
 import com.github.steveash.guavate.Guavate;
 import com.google.common.collect.ImmutableList;
 
-public class FilterToSearchQuery {
+public class
+
+
+
+
+
+FilterToSearchQuery {
```

Comment:
oops

Reply:
Sorry, since the last ssytem upgrade, archlinux is randmly adding some key strikes....